driver-core/src/main/java/com/datastax/driver/core/KeyspaceMetadata.java

```diff
@@ -51,6 +52,15 @@
         this.durableWrites = durableWrites;
         this.replication = replication;
         this.strategy = ReplicationStrategy.create(replication);
+        this.isVirtual=false;
```

Comment:
Would be preferable for this constructor should reuse the new one (i.e. `this(name, durableWrites, replication, false)`).

Should also mark this constructor as deprecated as it looks like existing uses of it were updated to use the new one.